### PR TITLE
Added env variable that enables/disables progress bar

### PR DESCRIPTION
**Pull request by manthony9**
Running node run-sign-addon.js n would disable the progress bar. Running it without any arguments would run the file with the progress bar. Added setProOptions function in PseudoProgress.js that assigns environment variable to local variable. A validation is performed in the show bucket function and executes the required action.

### CODE REVIEW

**Comment by Rob--W** [comment]
It looks like you're testing the functionality against a live demo server.
A unit test should ideally be self-contained and be independent of external dependencies.

Could you add a unit test? See https://github.com/mozilla/sign-addon/tree/master/tests for existing examples.

**Comment by manthony9** [comment]
@Rob--W Working on it. Will share an update as soon as I'm done.

**Comment by manthony9** [comment]
Hi @Rob--W, I tried making a small test in PseduoProgress.spec.ts not sure if it's the right approach, any feedback to improve would help. PseduoProgress.spec.js passes in general but import.spec.js fails. I haven't made any changes to import.spec . Snapshots below for your reference.

Failed Test
image
Passed Test
image
My addition to PseudoProgress.spec.ts
image

File: run-sign-addon.js [hunk 1]
```diff
@@ -0,0 +1,26 @@
+var { signAddon } = require('./dist/sign-addon.js');
```
Comment (willdurand) [inline run-sign-addon.js:1]: You should remove this file as well as the following ones:

- manifest.json
- cobster_loraboon_giraccork-1.0-an+fx.xpi
- cobster-loraboon-giraccork.zip

File: src/PseudoProgress.js [hunk 2]
```diff
@@ -42,13 +45,18 @@ class PseudoProgress {
     this.emptyBucketPointers = [];
 
     this.setPreamble(preamble);
+    this.setProOptions();
+  }
+
+  setProOptions() {
+    this.proOptions = process.argv[2];
```
Comment (willdurand) [inline src/PseudoProgress.js:52]: I think we want to pass this setting from the `signAddon` function. It could be a new parameter in the constructor.

**Review by willdurand — CHANGES_REQUESTED** [review]
That's a good start but we should take a slightly different direction.

**Comment by willdurand** [comment]
The issue has been closed by another patch.

**Pull request closed without merging**